cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -214,7 +233,9 @@ func (c *CStorVolumeController) cStorVolumeEventHandler(
 			cStorVolumeGot.Status.LastTransitionTime = cStorVolumeGot.Status.LastUpdateTime
 		}
 
-		cStorVolumeGot.Status.ReplicaStatuses = volStatus.ReplicaStatuses
+		if volStatus != nil {
```

Comment:
no need for this check

Reply:
It may be required if errror occurred for `istgtcontrol replica status` we are not returning(I don't know the reason) so instead of returning I made this check.